crates/chainspec/src/lib.rs

```diff
@@ -11,6 +11,12 @@
 
 extern crate alloc;
 
+use once_cell as _;
+#[cfg(not(feature = "std"))]
+pub(crate) use once_cell::sync::{Lazy as LazyLock, OnceCell as OnceLock};
+#[cfg(feature = "std")]
+pub(crate) use std::sync::{LazyLock, OnceLock};
+
```

Comment:
just use `extern crate alloc` and `alloc::sync::{LazyLock, OnceLock}`

see how other reth crates are doing it

Reply:
oh yeah, you're totally right